eman2/constants.py

```diff
@@ -212,3 +212,9 @@
 # SGD input types
 SGD_INPUT_AVG = 0
 SGD_INPUT_PTCLS = 1
+
+# Tomo
+TOMO_NEEDED_MSG = 'Tomography protocols couldn\'t be imported.\n' \
```

Comment:
Hi, I think @azazellochg agreed on setting a requirement for scipion-tomo. This way there is no need for importfromplugin and the message.

Reply:
true, I suggest to put ```install_requires=['scipion-em', 'scipion-em-tomo'],``` in setup.py

Reply:
Fixed in https://github.com/scipion-em/scipion-em-eman2/pull/64/commits/3b805869019181e3e364f9837a73ef99f09c12dd Currently, tomo protocols are inside a folder 'tomo_protocols'... not sure if we want to move them outside the folder. All of them already contain the word 'tomo' in the name so they can be easily identified...

Reply:
I don't have an opinion on whether we need a subfolder for tomo protocols, so either option is fine with me.

Reply:
I think this is up to the plugins maintainers...the submodule is plugin.protocols and every protocol importable from there. The internal files organization and convention is not strict, but of course, some consistency is good.

Reply:
Thanks guys! I have finally moved them to the parent folder (see https://github.com/scipion-em/scipion-em-eman2/pull/64/commits/5c621b2fde70aed199f265dcec6bc2d7eef2f8ea) No strong opinion either on my side but I feel it is handier if all protocols can be found at the same level